spec/factories/user.rb

```diff
@@ -1,31 +1,28 @@
 FactoryGirl.define do
   factory :user do
-    email "paullorijgu@gmail.com"
-    full_name  "Laul Guberson"
-    display_name "jgu"
-    password "gu"
-    role "default"
-    provider "twitter"
-    uid "abc123"
+    email "default.user@mail.com"
+    full_name  "Default user"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.